martinMutuma/def-politico
# #164734652  Add UI for upload an image

#### What does this PR do?
Enable a user to upload an image from local storage
#### Description of the task to be completed
Enable a user to upload an image from local storage
#### How should this be manually tested?
* Clone the repo `git clone https://github.com/martinMutuma/def-politico.git`
* Checkout to this branch `ft/164734652-ui-upload-image`
* Navigate to the profile page
* Click on the edit icon
* Notice a drop down box with the `upload an image button`
#### Any background context:
N/A
#### What are the relevant pivotal tracker stories?
#164734652
#### Screenshots
![image](https://user-images.githubusercontent.com/29132159/54664197-a14a7d80-4af4-11e9-8d8e-bfd428594439.png)
![image](https://user-images.githubusercontent.com/29132159/54664224-b2938a00-4af4-11e9-81bf-203504cd2cad.png)
